pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
+
+  src = fetchgit {
```

Comment:
I believe fetchFromGitHub only downloads less when there's a release available, which is not the case here:

```nix
  fetchFromGitHub = {
    owner, repo, rev, name ? "${repo}-${rev}-src",
    fetchSubmodules ? false,
    ... # For hash agility
  }@args:
  let
    baseUrl = "https://github.com/${owner}/${repo}";
    passthruAttrs = removeAttrs args [ "owner" "repo" "rev" "fetchSubmodules" ];
  in if fetchSubmodules then
    fetchgit ({
      inherit name rev fetchSubmodules;
      url = "${baseUrl}.git";
    } // passthruAttrs)
  else
    # We prefer fetchzip in cases we don't need submodules as the hash
    # is more stable in that case.
    fetchzip ({
      inherit name;
      url = "${baseUrl}/archive/${rev}.tar.gz";
      meta.homepage = "${baseUrl}/";
    } // passthruAttrs) // { inherit rev; };
```